catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -195,6 +195,10 @@ def __init__(
 
         self.ingestion_errors: list[IngestionError] = []  # Keep track of skipped errors
 
+        self._should_verbose_log = self.dag_id in Variable.get(
+            "SHOULD_VERBOSE_LOG", default_var=[], deserialize_json=True
+        )
```

Comment:
The name of the variable makes me think it should be a boolean. Do you have reasons to make `[]` the default?

```suggestion
        self._should_verbose_log = self.dag_id in Variable.get(
            "SHOULD_VERBOSE_LOG", default_var=False
        )
```

Reply:
Yes, @AetherUnbound told me at the PyCascades sprint she wanted to be able to activate the verbose logging per DAG (i.e. dag_id). This is the name we came up with. What naming would you propose?

Reply:
@krysal are you able to answer @nicoepp's question here?

Reply:
@nicoepp That's fine then! I wasn't sure how it would be used, but from what you say it makes sense.